src/content/en/fundamentals/accessibility/how-to-review.md

```diff
@@ -0,0 +1,269 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: How to review your site for accessibility issues.
+
+
+{# wf_updated_on: 2017-03-12 #}
+{# wf_published_on: 2017-03-12 #}
+
+# How To Do an Accessibility Review {: .page-title }
+
+{% include "web/_shared/contributors/robdodson.html" %}
+
+<div>
+  <div class="video-wrapper">
+    <iframe class="devsite-embedded-youtube-video" data-video-id="cOmehxAU_4s"
+            data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+    </iframe>
+  </div>
+Determining if your web site or application is accessible can seem like an
+overwhelming task. If you are approaching accessibility for the first time, the
+sheer breadth of the topic can leave you wondering where to start - after all,
+working to accommodate a diverse range of abilities means there are a
+correspondingly diverse range of issues to consider.
+
+In this post, I'm going to break down these issues into a logical, step by step
+process for reviewing an existing site for accessibility.
+</div>
+
+## Start with the keyboard
+<img src="imgs/ic_keyboard_black_24px.svg" class="attempt-right" alt="" width="120"/>
```

Comment:
No I purposefully used `alt=""` which removes the images from the screen reader flow. These images are primarily decorative so they don't offer much to a screen reader user.